src/components/arrow/arrow.tsx

```diff
@@ -1,5 +1,5 @@
-import React from 'react';
```

Comment:
I will just assume that all changes in `src` are eslint/prettier auto-fixes and not read those as thoroughly 🙈